tests/test_trackingstore.py

```diff
@@ -330,6 +330,60 @@ def test_create_run(mocker, mlflow_parent_run_id, faculty_parent_run_id):
     assert returned_run == mlflow_run
 
 
+def test_create_run_experiment_deleted(mocker):
+    mlflow_timestamp = mocker.Mock()
+    faculty_datetime = mocker.Mock()
+    timestamp_converter_mock = mocker.patch(
+        "mlflow_faculty.trackingstore."
+        "mlflow_timestamp_to_datetime_milliseconds",
+        return_value=faculty_datetime,
+    )
+
+    mlflow_tag = mocker.Mock()
+    faculty_tag = mocker.Mock()
+    tag_converter_mock = mocker.patch(
+        "mlflow_faculty.trackingstore.mlflow_tag_to_faculty_tag",
+        return_value=faculty_tag,
+    )
+
+    mock_client = mocker.Mock()
+    exception = faculty.clients.experiment.ExperimentDeleted(
+        message="message", experiment_id="test-id"
+    )
+
+    mock_client.create_run.side_effect = exception
+    mocker.patch("faculty.client", return_value=mock_client)
```

Comment:
I recommend adding an empty line below here and removing the empty line marked above to more clearly group the client mock setup lines together.